transforms/update-imports/README.md

```diff
@@ -0,0 +1,21 @@
+# update-imports
+
+
+## Usage
+
+```
+npx atam-codemod update-imports path/of/files/ or/some**/*glob.js
+
+# or
+
+yarn global add app-to-addon-migrator
+atam-codemod update-imports path/of/files/ or/some**/*glob.js
+```
+
+## Input / Output
+
+<!--FIXTURES_TOC_START-->
```

Comment:
I think you should run 
```
codemod-cli update-docs
```
to update the examples here

Reply:
Done